src/creature.h

```diff
@@ -142,7 +142,11 @@ class Creature
         /** Make a single melee attack with the currently equipped weapon against the targeted
          *  creature. Should always be overwritten by the appropriate player/NPC/monster function. */
         virtual void melee_attack(Creature &t, bool allow_special,
-                                  matec_id technique) = 0;
+                                  const matec_id & technique) = 0;
+        /**
+         * Calls the tother melee_attack function without any technique.
```

Comment:
It's definitely not a typo, I made it totally on purpose. To see whether somebody actually reads the comments. Yes, that's why it's there, not a type at all.